airlock/business_logic.py

```diff
@@ -1227,7 +1247,19 @@ def group_edit(
             extra={"context": context, "controls": controls},
         )
 
-        bll._dal.group_edit(release_request.id, group, context, controls, audit)
+        change_notifications = bll._dal.group_edit(
```

Comment:
Shouldn't this be `self._dal`? And same for a few other instances of this pattern that were in this file already before this diff.

Reply:
yes, it should - I just edited what was already there. I'll fix the others too